src/ui/camera.js

```diff
@@ -1588,6 +1647,22 @@ class Camera extends Evented {
             delta > 180 ? -360 :
             delta < -180 ? 360 : 0;
     }
+
+    // emulates frame function for some transform
+    _emulate(frame: Function, options: {duration: number, frameRate?: number, initialTransform?: Transform}) {
```

Comment:
Nit: we're not utilizing the flexibility of accepting options here — e.g. when we call this method, we always pass `initialTransform`, `frameRate` is always `15` — would it be simpler to just do `_emulate(frame, duration, initialTransform)` and hardcode the rest until it's needed?